pylint/utils/docs.py

```diff
@@ -42,27 +42,29 @@ def _get_checkers_infos(linter: PyLinter) -> dict[str, dict[str, Any]]:
     return by_checker
 
 
-def _get_checkers_documentation(linter: PyLinter) -> str:
+def _get_checkers_documentation(linter: PyLinter, show_options: bool = True) -> str:
     """Get documentation for individual checkers."""
-    result = get_rst_title("Pylint global options and switches", "-")
-    result += """
+    result = ""
+    if show_options:
```

Comment:
I was going to suggest a function but it's easier if I do the commit directly.

Reply:
(Beside we can check that the refactor is not changing the behavior easily as we have the full generated output in the repo)